validator/sawtooth_validator/journal/responder.py

```diff
@@ -93,13 +93,8 @@ def handle(self, connection_id, message_content):
         if block_request_message.nonce in self._seen_requests:
             LOGGER.debug("Received repeat GossipBlockRequest from %s",
                          connection_id)
-            ack = network_pb2.NetworkAcknowledgement()
-            ack.status = ack.OK
 
-            return HandlerResult(
-                HandlerStatus.RETURN,
-                message_out=ack,
-                message_type=validator_pb2.Message.NETWORK_ACK)
+            return HandlerResult(HandlerStatus.DROP)
```

Comment:
Should we use `PASS` for these?  Drop should only be used when we absolutely know no other handler in the handler chain should happen (e.g. the request is malformed).

Reply:
This is dropping a duplicate message we have already handled (no need to pass it along), so I thougt drop was fitting.